src/rf/apps/core/models.py

```diff
@@ -177,11 +187,49 @@ def to_json(self):
             'tile_origin': self.tile_origin,
             'resampling': self.resampling,
             'transparency': self.transparency,
-            'status': self.status,
+            'status_created': (True if self.status_created is not None
+                               else False),
+            'status_upload_start': (True if self.status_upload_start
+                                    is not None else False),
+            'status_upload_end': (True if self.status_upload_end is not None
+                                  else False),
```

Comment:
By using timestamps instead of booleans, it makes queries harder to read (eg. qs.filter(status_completed__isnull=False)`) and requires an extra step when converting to JSON. I don't think we use the timestamp anywhere except for figuring out the pending layers, but that is about to get removed. So maybe just turn them into bools?